static/app/views/automations/hooks/utils.tsx

```diff
@@ -12,3 +14,31 @@ export function useAutomationActions(automation: Automation): ActionType[] {
     ),
   ] as ActionType[];
 }
+
+export function useConnectedIds(storageKey: string, initialIds?: string[]) {
```

Comment:
Sorry for the nit but using an object argument is almost always better than positional params. Helps with readability when calling `useConnectedIds()` and also avoids needing to change the callsites if this is ever refactored to add more params in the future.

```suggestion
export function useConnectedIds({ storageKey, initialIds = [] }: UseConnectedIdsParams) {
```

Reply:
fixed 🫡 https://github.com/getsentry/sentry/pull/91408/commits/e93ac1c0779522d735b143f24d78009bc4316a9e